### ISSUE TITLE
3秒极速复刻 fastapi server.py的合成效果远不如webui.py

### ISSUE THREAD

**Issue by BeyondESH** [FunAudioLLM/CosyVoice#1661]
不知道为什么3秒极速复刻用fastapi server.py的合成效果远不如直接用webui.py
生成的音频还原度高。

查看源码看到webui.py多了一个音频处理函数，将其加入到 fastapi server.py里效果还是不行，不知道为什么
def postprocess(speech, top_db=60, hop_length=220, win_length=440):
    # 1. 去除音频首尾静音
    speech, _ = librosa.effects.trim(
        speech, top_db=top_db,
        frame_length=win_length,
        hop_length=hop_length
    )
    # 2. 音量归一化
    if speech.abs().max() > max_val:
        speech = speech / speech.abs().max() * max_val
    # 3. 添加尾部静音防止截断
    speech = torch.concat([speech, torch.zeros(1, int(cosyvoice.sample_rate * 0.2))], dim=1)
    return speech

**Comment by BeyondESH** [comment]
找到原因了，是pcm流转wav时采样率过低的问题
